man/org.freedesktop.portable1.xml

```diff
@@ -0,0 +1,427 @@
+
+    <variablelist class="dbus-interface" generated="True" extra-ref="org.freedesktop.portable1.Manager"/>
+
+    <variablelist class="dbus-interface" generated="True" extra-ref="org.freedesktop.portable1.Manager"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="GetImage()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="ListImages()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="GetImageOSRelease()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="GetImageMetadata()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="GetImageState()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="AttachImage()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="DetachImage()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="ReattachImage()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="RemoveImage()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="MarkImageReadOnly()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="SetImageLimit()"/>
+
+    <variablelist class="dbus-method" generated="True" extra-ref="SetPoolLimit()"/>
+
+    <variablelist class="dbus-property" generated="True" extra-ref="PoolPath"/>
+
+    <variablelist class="dbus-property" generated="True" extra-ref="PoolUsage"/>
+
+    <variablelist class="dbus-property" generated="True" extra-ref="PoolLimit"/>
+
+    <variablelist class="dbus-property" generated="True" extra-ref="Profiles"/>
+
+    <!--End of Autogenerated section-->
+
+    <refsect2>
+      <title>Methods</title>
+
+      <para><function>GetImage()</function> may be used to get the image object path of the image with the
+      specified name.</para>
+
+      <para><function>ListImages()</function> returns an array of all currently known images. The
+      structures in the array consist of the following fields: image name, type, read-only flag, creation
+      time, modification time, current disk space, usage and image object path.</para>
+
+      <para><function>GetImageOSRelease()</function> retrieves the OS release information of an image.
+      This method returns an array of key value pairs read from the
+      <citerefentry><refentrytitle>os-release</refentrytitle><manvolnum>5</manvolnum></citerefentry> file in
+      the image and is useful to identify the operating system used in a portable service.</para>
+
+      <para><function>GetImageMetadata()</function> retrieves metadata associated with an image.
+      This method returns the image name, the image's <citerefentry><refentrytitle>os-release</refentrytitle>
+      <manvolnum>5</manvolnum></citerefentry> content in the form of a (streamable) array of bytes,
+      and a list of each portable unit contained in the image, in the form of a string
+      (unit name) and a streamable array of byte with the content.</para>
```

Comment:
an a a list of each portable unit → and a list of portable units

What does "streamable" mean in this context? Doesn't this just send a dbus message with fixed contents?

Reply:
It was supposed to be a hint for implementers, and describe how portablectl consumes it, but I'll drop it since it can be confusing